pyramid/metrics.go

```diff
@@ -0,0 +1,41 @@
+package pyramid
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// nolint: gomnd
+const (
+	kb          = float64(1024)
+	mb          = 1024 * kb
+	fsNameLabel = "fsName"
+)
+
+var cacheHit = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "tier_fs_cache_hits_total",
+		Help: "TierFS cache hits total count",
+	}, []string{fsNameLabel})
+
+var cacheMiss = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "tier_fs_cache_miss_total",
+		Help: "TierFS cache miss total count",
+	}, []string{fsNameLabel})
```

Comment:
I very much prefer to monitor caches with a single metric "access", that has a "status" label "hit" or "miss".  That makes displaying miss percentages much easier, because no need to join the streams to get a total.